WalletWasabi.Fluent/Views/CoinJoinProfiles/CoinJoinProfilesView.axaml

```diff
@@ -0,0 +1,91 @@
+<UserControl xmlns="https://github.com/avaloniaui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+             xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+             xmlns:vm="using:WalletWasabi.Fluent.ViewModels.CoinJoinProfiles"
+             xmlns:c="using:WalletWasabi.Fluent.Controls"
+             xmlns:conv="using:WalletWasabi.Fluent.Converters"
+             xmlns:prof="clr-namespace:WalletWasabi.Fluent.Views.CoinJoinProfiles"
+             mc:Ignorable="d" d:DesignWidth="428" d:DesignHeight="371"
+             x:DataType="vm:CoinJoinProfilesViewModel"
+             x:CompileBindings="True"
+             x:Class="WalletWasabi.Fluent.Views.CoinJoinProfiles.CoinJoinProfilesView">
+  <c:ContentArea Title="{Binding Title}" Caption=""
+                 EnableCancel="{Binding EnableCancel}"
+                 EnableBack="{Binding EnableBack}"
+                 NextContent="Continue" EnableNext="True"
+                 ScrollViewer.VerticalScrollBarVisibility="Disabled">
+    <DockPanel>
+
+      <TextBlock DockPanel.Dock="Top" Text="Wasabi takes care of your financial privacy by automatically coinjoining all your funds for a 0.3% coordination fee and for variable Bitcoin network fees. Select a coinjoin strategy that fits you best!" TextWrapping="Wrap" />
```

Comment:
Do we consistantly not capitalize `coinjoin`?

Reply:
It seems like https://github.com/zkSNACKs/WalletWasabi/pull/7047